src/generic/grid.cpp

```diff
@@ -3864,6 +3866,10 @@ void wxGrid::ProcessRowLabelMouseEvent( wxMouseEvent& event, wxGridRowLabelWindo
     int y;
     wxGridWindow *gridWindow = rowLabelWin->IsFrozen() ? m_frozenRowGridWin : m_gridWin;
 
+    // store position, before its modified in the next step
+    int event_y = event.GetPosition().y;
+    static int last_event_y;
```

Comment:
I guess we could make this static because you can't drag-resize more than one grid at the same time, but I'd still prefer to make it a member variable instead, why should it be different from all the rest of drag-related data?

Reply:
OK, will do so in a later commit.